azkaban-common/src/main/java/azkaban/executor/JdbcExecutorLoader.java

```diff
@@ -479,6 +479,25 @@ public void updateExecutableNode(ExecutableNode node)
     }
 
     QueryRunner runner = createQueryRunner();
+      if (node.getAttempt() != 0) {
+        List<ExecutionAttempt> past = node.getPastAttemptList();
+        for (ExecutionAttempt executionAttempt : past) {
+          try {
+             runner.update(
+             "UPDATE execution_jobs SET start_time=?, end_time=?, status=?  WHERE exec_id=? AND flow_id=? AND job_id=? AND attempt=?" , 
```

Comment:
Nit: Since each of these lines are part of the method call on L486, indent each of these lines in by 4 spaces.